### PR TITLE
improved readme

### PR DESCRIPTION
**Pull request by rutikpawar246**
## Purpose / Description
i am new contributor, i liked your repository, wanted to do first contribution,

## Fixes
Fixes _Link to the issues._

## Approach
I improved the readme file a little bit

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,4 +49,3 @@ See the [contribution wiki page](https://github.com/Akshay0701/food_delivery_app
 License
 -------
 [GPL-3.0 License](https://github.com/ankidroid/Anki-Android/blob/master/COPYING)  
-
```
Comment (Akshay0701) [inline README.md:51]: is there any reason for removing this line?

File: README.md [hunk 2]
```diff
@@ -16,17 +17,17 @@ A new Flutter project
 
 - Simple UI for food delivery app's
 
-- use SQLite for store order data in local database 
+- Use SQLite to store order data in a local database
```
Comment (Akshay0701) [inline README.md:20]: the previous sentence was also right, 

but anyway Thanks for the improvement!

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Food Delivery App
 ![](banner.png)
-A new Flutter project
+
+The food ordering & delivery application is designed based on the real-life scenario. Anyone can use this code for their own purpose. If you are interested please make your contribution to the code.
```
Comment (Akshay0701) [inline README.md:4]:
```suggestion
This is a food ordering & delivery application built with a flutter framework and firebase real-time database. 
Anyone can use this code for their own purpose. If you are interested please make your contribution to the code.
```

IMO we should provide information about the framework used to build this application,

What do you say?

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
Thank you so much @rutikpawar246,

I am really glad this is your first contribution, 

I mentioned one nit and question about line deletion.

File: README.md (hunk 1 above)
Comment (rutikpawar246) [reply README.md:51]: actually this line is not necessary in the documentation at end, so i removed it

File: README.md (hunk 3 above)
Comment (rutikpawar246) [reply README.md]: I agreed

**Comment by rutikpawar246** [comment]
Done @Akshay0701 :)

**Review by Akshay0701 — APPROVED** [review]
LGTM,
Thank you @rutikpawar246.

Merging now

**Pull request merged**